composer.json

```diff
@@ -6,36 +6,49 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
-        "drupal/core": "^9.3",
+        "php": ">=8.0",
+        "drupal/core": "^9.4",
```

Comment:
We require the PHP version based on Drupal version not the other way around and Drupal 9.x still works with PHP 7.4 so we don't need to add this in the readme, it's an actual [Drupal requirement](https://www.drupal.org/docs/system-requirements/php-requirements) not something specific to this project.

However, the core version update in the .info files is missing, I'll fix that.

Reply:
Updated the core recommended version in info files.

Reply:
> We require the PHP version based on Drupal version not the other way around and Drupal 9.x still works with PHP 7.4 so we don't need to add this in the readme, it's an actual [Drupal requirement](https://www.drupal.org/docs/system-requirements/php-requirements) not something specific to this project.

I meant so that developers in coming PRs know why we did it

Reply:
We never add info to the readme for why we update the PHP or the Drupal core version. I don't consider this necessary, sorry.

Reply:
I wanted something in the PR description at least but it's ok, I'll leave a reviewer comment so next person sees and knows what it was about.